html/syntax/serializing-html-fragments/initial-linefeed-pre.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<title>innerHTML getter for pre/textarea/listing with initial LF</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="outer">
+<div id="inner">
+<pre id="pre1">
+x</pre>
+<pre id="pre2">
+
+x</pre>
+<textarea id="textarea1">
+x</textarea>
+<textarea id="textarea2">
+
+x</textarea>
+<listing id="listing1">
+x</listing>
+<listing id="listing2">
+
+x</listing>
+</div>
+</div>
+
+<script id="expected" type="text/html">
+<div id="inner">
+<pre id="pre1">x</pre>
+<pre id="pre2">
+
+x</pre>
+<textarea id="textarea1">x</textarea>
+<textarea id="textarea2">
+
+x</textarea>
+<listing id="listing1">x</listing>
+<listing id="listing2">
+
+x</listing>
+</div>
+</script>
+
+<script>
+var expected_outer = expected.textContent;
```

Comment:
I feel like we should have the expected string explicit here, rather than relying on textContent being correct.